### PR TITLE
Update --perf config to latest

### PR DESCRIPTION
**Pull request by paulirish**
once we have tags in the default config (#1463), we can swap to that and won't have it fall behind again.

but until then..

### CODE REVIEW

File: lighthouse-core/config/perf.json [hunk 1]
```diff
@@ -12,7 +26,10 @@
     "time-to-interactive",
     "user-timings",
     "screenshots",
-    "critical-request-chains"
+    "critical-request-chains",
+    "unused-css-rules",
+    "dobetterweb/uses-optimized-images",
+    "dobetterweb/uses-responsive-images"
```
Comment (ebidel) [inline lighthouse-core/config/perf.json]: should we add these audits as well:

links-blocking-first-paint
scripts-blocking-first-paint

Comment (paulirish) [inline lighthouse-core/config/perf.json]: yes thanks!

Comment (paulirish) [inline lighthouse-core/config/perf.json]: added

**Comment by paulirish** [comment]
PTAL

Reports look like this:

image
image

File: lighthouse-core/config/perf.json [hunk 2]
```diff
@@ -2,8 +2,23 @@
   "passes": [{
     "recordNetwork": true,
     "recordTrace": true,
-    "gatherers": []
-  }],
+    "gatherers": [
+      "url",
+      "image-usage",
```
Comment (brendankenny) [inline lighthouse-core/config/perf.json]: doesn't look like it will matter, but any reason to include these in the first pass instead of the second?

Comment (paulirish) [inline lighthouse-core/config/perf.json]: nope.

**Pull request merged**